test/native/setup.js

```diff
@@ -154,6 +154,17 @@ jest.mock( 'react-native-reanimated', () => {
 // https://reactnavigation.org/docs/testing/#mocking-native-modules
 jest.mock( 'react-native/Libraries/Animated/NativeAnimatedHelper' );
 
+/**
+ * Note: Clipboard has been extracted from react-native core and will be removed in a future release.
+ * It can now be installed and imported from @react-native-community/clipboard instead of 'react-native'.
+ *
+ * @see node_modules/react-native/Libraries/Components/Clipboard/Clipboard.js
+ */
+jest.mock( 'react-native/Libraries/Components/Clipboard/Clipboard', () => ( {
```

Comment:
@ttahmouch Looks like the Clipboard library is also mocked line 189, I recall I also had to add it for other integration tests. I don't think this would cause any conflict but might be confussing having duplicated code, wdyt?

Reply:
Thanks for letting me know, @fluiddot . I think I remember you adding it for work related to the Embed Block that you and Joel implemented.

I think this may have slipped by when I merged the `trunk` branch downstream into my feature branch to keep it up-to-date before release. I can remove the redundancy.

Reply:
No worries, it happened to me several times 😅 . I wish Git could detect this redundancy cases although they are not trivial at all to infer.